subset/security/tlsfaux/server.py

```diff
@@ -1,17 +1,17 @@
 import socket, ssl, pprint
 
-hostname='0.0.0.0'
-port=443
+hostname = '0.0.0.0'
+port = 443
 
-filenameCert='certs/server.crt'
-filenameKey='certs/server.key'
+filename_cert = 'certs/server.crt'
+filename_key = 'certs/server.key'
 
 context = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
-context.load_cert_chain(certfile=filenameCert, keyfile=filenameKey)
+context.load_cert_chain(certfile = filename_cert, keyfile = filename_key)
```

Comment:
Should run pylint.  The style is weird, for this use (keyword values), there is no space around =, while for variable assignments there is. Go figure.